### DISCUSSION TITLE
Welcome to the ARMI Discussions!

### DISCUSSION THREAD

**Discussion by ntouran** [terrapower/armi#279] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ntouran** [comment]
I'm Nick. I have been working on ARMI since 2009, and on ARMI's predecessors since 2006 when I first started running scripts that automated runs of MC2-2 and REBUS fast reactor analysis codes. I'm a nuclear engineer focused on making the process of reactor development and deployment better (in terms of cost, quality, and speed). In my spare time I walk my dog and work on my personal website, [whatisnuclear.com](https://whatisnuclear.com) (where [this is my favorite contribution there](https://whatisnuclear.com/reactor_history.html)).

**Comment by drewejohnson** [comment]
This is excellent. Very much on board for discussions :rocket: 

I'm Drew, recently started working with ARMI at [USNC Technologies](https://usnc.com/space/). Been a fan of open source nuclear for a few years, developing [`serpentTools`](https://serpent-tools.readthedocs.io/en/latest/) for parsing Serpent output files in Python and contributing to [`openmc` ](https://openmc.org/)

**Comment by jakehader** [comment]
Hi - I'm Jake and I've been working at TerraPower for just over 5 years now. My interest are in reactor physics, specifically with fast reactors, and in radiation shielding. I enjoy working with nuclear data and would love to contribute to any improvements on nuclear data management internal or external to the ARMI framework. 

I am familiar with fast reactor design codes, including MC2-3, DIF3D/VARIANT, MCNP, PARTISN, and PERSENT. I have some limited experience with light water reactor cross section codes like CASMO and HELIOS. Feel free to reach out to me anytime!

**Comment by john-science** [comment]
I'm John and I have been working on the core ARMI team at TerraPower for about 6 months. I have graduate degrees in Physics and Engineering, but neither of those in Nuclear Engineering.  i am the Senior Software Engineer at TP and split my time between writing code and supporting nuclear engineers who are working in the ARMI ecosystem.

**Reply by john-science** [reply]
@scottyak   Feel free to say "hi"!

**Comment by scottyak** [comment]
Hi all, I am Scott. I am a software engineer currently working at Google. I studied Physics for a 1 year or so before switching to CS in college, and have no background in nuclear engineering. I am not affiliated with Terrapower -- I am just a fan of nuclear energy who wants to get involved in some way in my (unfortunately rather limited) personal time, and I hope to fix some bugs, spread the word about this project, and get more people involved :)
